apps/ledger-live-desktop/tests/specs/services/cosmosStaking.spec.ts

```diff
@@ -0,0 +1,140 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { Drawer } from "../../models/Drawer";
+import { Modal } from "../../models/Modal";
+import { PortfolioPage } from "tests/models/PortfolioPage";
+import { Layout } from "tests/models/Layout";
+import { AccountPage } from "tests/models/AccountPage";
+
+test.use({
+  userdata: "cosmosOsmosisStakingAccounts",
+  featureFlags: {
+    stakePrograms: {
+      enabled: true,
+      params: {
+        list: ["ethereum", "solana", "tezos", "polkadot", "tron", "cosmos", "osmo", "celo", "near"],
+      },
+    },
+    portfolioExchangeBanner: {
+      enabled: true,
+    },
+    stakeAccountBanner: {
+      enabled: true,
+      params: {
+        solana: {
+          redelegate: true,
+          delegegate: true,
+        },
+        eth: {
+          kiln: true,
+          lido: true,
+        },
+      },
+    },
+  },
+});
+
+test("Cosmos staking flows via portfolio, asset page and market page", async ({ page }) => {
```

Comment:
Might be worth adding a test or two to make sure the staking banner/buttons are displayed in the Cosmos account screen @sarneijim

Reply:
Banner screenshots are done, perhaps the title is not accurate